requirements.txt

```diff
@@ -1,18 +1,69 @@
-dateparser
-django
-django-elasticsearch-dsl
-django-environ
-gunicorn
-requests
-beautifulsoup4
-mixer
-psycopg2-binary
-pytest
-pytest-django
-pytest-cov
+argcomplete==1.10.0
```

Comment:
Uh… I have to show you Pipenv. https://pipenv.readthedocs.io/en/latest/

Reply:
Can you please stick to only the high-level dependencies you've explicitly added yourself in this file. You can pin them if you want, but don't include the dependencies of dependencies. Otherwise we're very soon going to have chaos where we don't know what you or I have added ourselves and what's an implicit dependency that a library has added.

Reply:
You're right. Marko also advised me to do the same.